examples/IBLevelSet/ex6/example.cpp

```diff
@@ -0,0 +1,626 @@
+                "u_init", app_initializer->getComponentDatabase("VelocityInitialConditions"), grid_geometry);
+            navier_stokes_integrator->registerVelocityInitialConditions(u_init);
+        }
+
+        if (input_db->keyExists("PressureInitialConditions"))
+        {
+            Pointer<CartGridFunction> p_init = new muParserCartGridFunction(
+                "p_init", app_initializer->getComponentDatabase("PressureInitialConditions"), grid_geometry);
+            navier_stokes_integrator->registerPressureInitialConditions(p_init);
+        }
+
+        // Create Eulerian boundary condition specification objects (when necessary).
+        const IntVector<NDIM>& periodic_shift = grid_geometry->getPeriodicShift();
+        vector<std::unique_ptr<RobinBcCoefStrategy<NDIM> > > u_bc_coefs(NDIM);
+        if (periodic_shift.min() > 0)
+        {
+            for (unsigned int d = 0; d < NDIM; ++d)
+            {
+                u_bc_coefs[d] = nullptr;
+            }
+        }
+        else
+        {
+            for (unsigned int d = 0; d < NDIM; ++d)
+            {
+                ostringstream bc_coefs_name_stream;
+                bc_coefs_name_stream << "u_bc_coefs_" << d;
+                const string bc_coefs_name = bc_coefs_name_stream.str();
+
+                ostringstream bc_coefs_db_name_stream;
+                bc_coefs_db_name_stream << "VelocityBcCoefs_" << d;
+                const string bc_coefs_db_name = bc_coefs_db_name_stream.str();
+
+                u_bc_coefs[d] = std::make_unique<muParserRobinBcCoefs>(
+                    bc_coefs_name, app_initializer->getComponentDatabase(bc_coefs_db_name), grid_geometry);
+            }
+            navier_stokes_integrator->registerPhysicalBoundaryConditions({
+                u_bc_coefs[0].get(), u_bc_coefs[1].get()
+#if (NDIM == 3)
+                                         ,
+                    u_bc_coefs[2].get()
+#endif
+            });
+        }
+
+        std::unique_ptr<RobinBcCoefStrategy<NDIM> > rho_bc_coef = nullptr;
+        if (!(periodic_shift.min() > 0) && input_db->keyExists("DensityBcCoefs"))
+        {
+            rho_bc_coef = std::make_unique<muParserRobinBcCoefs>(
+                "rho_bc_coef", app_initializer->getComponentDatabase("DensityBcCoefs"), grid_geometry);
+            navier_stokes_integrator->registerMassDensityBoundaryConditions(rho_bc_coef.get());
+        }
+
+        std::unique_ptr<RobinBcCoefStrategy<NDIM> > mu_bc_coef = nullptr;
+        if (!(periodic_shift.min() > 0) && input_db->keyExists("ViscosityBcCoefs"))
+        {
+            mu_bc_coef = std::make_unique<muParserRobinBcCoefs>(
+                "mu_bc_coef", app_initializer->getComponentDatabase("ViscosityBcCoefs"), grid_geometry);
+            navier_stokes_integrator->registerViscosityBoundaryConditions(mu_bc_coef.get());
+        }
+
+        std::unique_ptr<RobinBcCoefStrategy<NDIM> > phi_bc_coef = nullptr;
```

Comment:
```suggestion
        std::unique_ptr<RobinBcCoefStrategy<NDIM> > phi_bc_coef;
```

Reply:
I have made the requested changes. Please take a look.